optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -34,6 +34,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-test-util</artifactId>
```

Comment:
Perfect - not sure why I missed that when glancing over the drools repo.
This is arguably better than a test-jar dependency as the PersistenceUnit is a common thing to reuse (I 've just seen copies of it in droolsjbpm-integration repo, but we can worry about that later).